msprime/trees.py

```diff
@@ -327,7 +323,10 @@ def draw(self):
         for u in self._tree.nodes():
             v = self._tree.get_parent(u)
             x = self._x_coords[u], self._y_coords[u]
-            dwg.add(dwg.circle(center=x, r=3))
+            if u in self._node_colours and self._node_colours[u] is not None:
```

Comment:
Why are we comparing with ``None`` here? The input map is from nodes to colours.